docs/navigatorios.md

```diff
@@ -3,6 +3,8 @@ id: navigatorios
 title: NavigatorIOS
```

Comment:
Should we just delete this file?

Reply:
I thought this would cause issues with Docusaurus, but after testing it seems fine, as long as the entry is removed from sidebars.json.

I'll pull in this pull request to get the changes to docs/navigation.md in, and I'll remove navigatorios.md in a separate PR.